torch/package/importer.py

```diff
@@ -184,6 +184,13 @@ class OrderedImporter(Importer):
     def __init__(self, *args):
         self._importers: List[Importer] = list(args)
 
+    def _check_if_malformed_package(self, module):
+        if not hasattr(module, '__path__'):
```

Comment:
https://docs.python.org/3/reference/import.html#path__ mentions `__path__` has to be set for packages, so should this return True? or is it that it returns False because without `__path__` the module is not a package at all, rather just a module?

Reply:
ohh shoot just bad naming on my end. Maybe malformed module would be better?

Reply:
Well if it's not a package, it definitely can't be a malformed package which was my logic here haha. But yeah we do need a better name, I'll just go with fileless_package for now